app/serializers/word.rb

```diff
@@ -0,0 +1,5 @@
+class WordSerializer
```

Comment:
I am not understanding the need for a WordSerializer as we never the intent/goal (as I understand it) is to respond with the GamePlaySerializer. 

- Is this leftover from the early days of the project and no longer needed?

Reply:
Most likely, we don't need this. I believe @jlweave wrote this, I updated the gem call on line from:
include JSON::Serializer
to 
include JSONAPI::Serializer
Realistically, I don't think we need a word serializer since the GamePlaySerializer is returning all of the words we need

Reply:
can we delete this loose end?